.expeditor/verify.pipeline.yml

```diff
@@ -7,7 +7,7 @@ expeditor:
       retry:
         automatic:
           limit: 1
-      timeout_in_minutes: 45
+      timeout_in_minutes: 90
```

Comment:
I thought 60 was the limit?

Reply:
Oh I missed it to revert it back. Thanks! 
Done.